veripy/steps/content/checks/element_contains_text.py

```diff
@@ -18,3 +17,39 @@ def check_element_text(context, element_name, text):
         f'The "{element_name}" was supposed to contain "{text}", '
         f'but instead had "{page_element.text}".'
     )
+
+
+@then('the following {colname} are {not_:optional_not}displayed')
+def check_elements_have_text(context, colname, not_):
+    """ Asserts that the element is the visible on the page.
+    """
+    if getattr(context, 'table') is None:
+        raise AssertionError('No items were defined.')
+    if not context.table.has_column('element'):
+        raise AssertionError(f'The table does not have a header \'element\'.')
+
+    if not context.table.has_column(colname):
+        raise AssertionError(f'The table does not have a header \'{colname}\'.')
```

Comment:
> When a string contains single or double quote characters, however, use the other one to avoid backslashes in the string. It improves readability.

* https://pep8.org/#string-quotes